src/api_test.ts

```diff
@@ -140,22 +131,19 @@ Deno.test("SlackAPI class", async (t) => {
               });
             });
 
-            await assertRejects(
-              async () => {
-                return await client.response(
-                  "https://slack.com/api/chat.postMessage",
-                  {},
-                );
-              },
-              (error: Error) => {
-                assertInstanceOf(error, HttpError);
-                if (isHttpError(error)) {
-                  assertEquals(error.headers?.get("Retry-After"), "120");
-                  assertEquals(error.status, 429);
-                  assertEquals(error.message, "429: ratelimited");
-                }
-              },
-            );
+            try {
+              await client.response(
```

Comment:
What do you think about adding the following, at the end of the `try` block, in order to endure that en error is always thrown

```ts
assert(false, "An error should have been thrown in this try block");
```

Reply:
Good catch! I will add that